api/v1beta1/operatorpolicy_types.go

```diff
@@ -134,6 +134,13 @@ type ComplianceConfig struct {
 	//
 	//+kubebuilder:default=Compliant
 	UpgradesAvailable ComplianceConfigAction `json:"upgradesAvailable,omitempty"`
+	// DeprecationAvailable specifies how the NoDeprecations condition impacts overall
+	// policy compliance. The default value is `Compliant`. If any deprecations are detected
+	// in the package, channel, or bundle with DeprecationAvailable = NonCompliant,
+	// then the policy compliance will be set to `Compliant`.
+	//
+	// +kubebuilder:default=Compliant
+	DeprecationAvailable ComplianceConfigAction `json:"deprecationAvailable,omitempty"`
```

Comment:
Maybe `DeprecationsPresent` would make this more clear?

```suggestion
	// DeprecationsPresent specifies how the overall policy compliance is affected by deprecations.
	// The default value is `Compliant`. If any deprecations are detected
	// while DeprecationsPresent = NonCompliant,
	// then the policy compliance will be set to `NonCompliant`.
	//
	// +kubebuilder:default=Compliant
	DeprecationsPresent ComplianceConfigAction `json:"deprecationsPresent,omitempty"`
```

Reply:
genius